css/pages/card/card.html

```diff
@@ -0,0 +1,283 @@
+                    </iframe>
+                </div>
+            </div>
+        </div>
+        <div id="card_heading_on_top_image">
+            <h3 class="main_container_heading fs-m py-x-small">Heading on Top of Image</h3>
+            <p class="main_container_description py-x-small">This Type of Card is same as above card with a slight change i.e now the heading is
+                over the image.
+            </p>
+            <div>
+                <div class="preview_container p-x-small">
+                    <div class="card card_header">
+                        <div class="card_content">
+                            <div class="card_content_header">
+                                <div class="primary_header">
+                                    Valorant Gaming
+                                </div>
+                                <div class="secondary_header">
+                                    By Swapnil Bansal
+                                </div>
+                            </div>
+                            <img class="card_content_image" src="../../vendors/card_image.png" , alt="">
+                            <div class="card_content_body">
+                                Leh is the joint capital and largest town of the union territory of Ladakh in India.
+                                Leh, located in the
+                                Leh district, was also the historical capital of the Kingdom of Ladakh,
+                            </div>
+                        </div>
+                        <div class="card_actions">
+                            <div class="card_button_action">
+                                <button class="btn_card">READ</button>
+                                <button class="btn_card">BOOKMARK</button>
+                            </div>
+                            <div class="card_icon_action">
+                                <button class="btn_card_icon"><i class="fas fa-bookmark fa-lg"></i></button>
+                                <button class="btn_card_icon"><i class="fas fa-share fa-lg"></i></button>
+                                <button class="btn_card_icon"><i class="fas fa-ellipsis-v fa-lg"></i></button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div>
+                    <iframe src="https://www.thiscodeworks.com/embed/61fa9e08b783be0015bbaee4"
+                        style="width: 100%; height: 463px;" frameborder="0">
+                    </iframe>
+                </div>
+            </div>
+        </div>
+        <div id="card_horizontal">
+            <h3 class="main_container_heading fs-m py-x-small">Horizontal Card</h3>
+            <p class="main_container_description py-x-small">Horizontal Card are those card which has width greater than its height.
+                Below is the example of the Horizontal Card.This card does not contain can space for any content.
```

Comment:
typo: add space after dot(.)

Reply:
it is because of formatting.Not effecting the output